Source/Filter/BilinearFilter.cpp

```diff
@@ -20,8 +20,8 @@ using namespace amrex;
 namespace {
     void compute_stencil(Gpu::DeviceVector<Real> &stencil, int npass)
```

Comment:
The better solution to this would be to express the allowed value range of `npass`, which is `unsigned int`

Reply:
Sure! I didn't do that because I wasn't sure that we were ok with changing the signature. But I totally agree that it is the right thing to do here...